inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -529,4 +323,51 @@ void MKLDNNSplitNode::setDynamicBatchLim(int lim) {
         prim.setBatchLimit(batchToProcess(), getParentEdges().size(), getChildEdges().size());
     }
 }
+
+inline uint8_t* MKLDNNSplitNode::getDataPtr(const MKLDNNMemory& memoryPtr) {
+    return reinterpret_cast<uint8_t*>(memoryPtr.GetData()) + memoryPtr.GetDescriptor().data.layout_desc.blocking.offset_padding *
+            MKLDNNExtensionUtils::sizeOfDataType(mkldnn::memory::data_type(memoryPtr.GetDescriptor().data.data_type));
+}
+
+void MKLDNNSplitNode::prepareOptimizedParams() {
+    auto inpTensorDesc = this->getSelectedPrimitiveDescriptor()->getConfig().inConfs[0].desc;
+
+    //find axis order position
+    auto order = inpTensorDesc.getBlockingDesc().getOrder();
+    unsigned axisOrderPos = UINT_MAX;
+    for (size_t i = 0; i < order.size(); ++i) {
+        if (order[i] == axis) {
+            axisOrderPos = i;
+            break;
+        }
+    }
+    if (UINT_MAX == axisOrderPos) {
+        THROW_IE_EXCEPTION << "Can't find the axis in the input tensor order list";
+    }
+
+    uint8_t srcDataSize = inpTensorDesc.getPrecision().size();
+    auto srcDims = inpTensorDesc.getBlockingDesc().getBlockDims();
+    int nDims = srcDims.size();
+
+    optimizedParams.countStrides = 1;
+    for (int i = 0; i < axisOrderPos; i++)
+            optimizedParams.countStrides *= srcDims[i];
```

Comment:
Broken alignment?

Reply:
Fixed in ee8234d71967059ae0d72418abd7796120119305.